experimental/bindings/java/build.gradle

```diff
@@ -54,8 +58,30 @@ android {
 
     externalNativeBuild {
         cmake {
-            version "3.13.0+"
```

Comment:
Why change to 3.18.4? We should match https://github.com/google/iree/blob/c67112d825feb8a8c820756f24ce6200028d2137/CMakeLists.txt#L15 unless there is a specific reason to require a higher version for gradle.

Reply:
This was a mistake add. Reverted